extensions/objects/models/objects.py

```diff
@@ -824,3 +824,23 @@ class ListOfSetsOfHtmlStrings(BaseObject):
         'type': 'list',
         'items': SetOfHtmlString.SCHEMA,
     }
+
+
+class DragAndDropHtmlString(BaseObject):
+    """A drag and drop Html string class."""
+
+    description = 'A drag and drop Html string.'
+    default_value = ''
+
+    SCHEMA = {
+        'type': 'html'
+    }
+
+
+class DragAndDropPositiveInt(BaseObject):
+    """A drag and drop positive int class."""
+
+    description = 'A drag and drop positive int.'
```

Comment:
A drag and drop item rank/position in the collection of drag and drop items?

Reply:
Or, `The rank(position) of a drag and drop item in the given set of list of drag and drop items`?

Reply:
Done